test/API/endpoints/CloudRecording.spec.js

```diff
@@ -0,0 +1,219 @@
+      .listMeetingRecordings({ meetingId: 12345 });
+    assert.equal(
+      ret.path,
+      '/meetings/12345/recordings',
+      'path does not match'
+    );
+    assert.equal(
+      ret.method,
+      'GET',
+      'method does not match'
+    );
+    assert.deepEqual(
+      ret.params,
+      undefined,
+      'params should be empty'
+    );
+  });
+
+  it('lists Meeting Recordings with double encoding needed (Test 1)', async function () {
+    const ret = await testAPI.cloudRecording.listMeetingRecordings({ meetingId: '/xUPpM3lTxqFSV4bVBrzDQ==' });
+    assert.equal(
+      ret.path,
+      '/meetings/%252FxUPpM3lTxqFSV4bVBrzDQ%253D%253D/recordings',
+      'double encoding failed'
+    );
+  });
+
+  it('lists Meeting Recordings with double encoding needed (Test 2)', async function () {
+    const ret = await testAPI.cloudRecording.listMeetingRecordings({ meetingId: 'XA+DOiHeR8+Cs/5vf2eJGQ==' });
+    assert.equal(
+      ret.path,
+      '/meetings/XA%252BDOiHeR8%252BCs%252F5vf2eJGQ%253D%253D/recordings',
+      'double encoding failed'
+    );
+  });
+
+  it('lists Meeting Recordings with double encoding needed (Test 3)', async function () {
+    const ret = await testAPI.cloudRecording.listMeetingRecordings({ meetingId: 'XZuBo//2QQKmm/v2+Cxwvw==' });
+    assert.equal(
+      ret.path,
+      '/meetings/XZuBo%252F%252F2QQKmm%252Fv2%252BCxwvw%253D%253D/recordings',
+      'double encoding failed'
+    );
+  });
+
+  it('lists User Recordings with only required params', async function () {
+    const ret = await testAPI.cloudRecording
+      .listUserRecordings({ userId: '12345' });
+    assert.equal(
+      ret.path,
+      '/users/12345/recordings',
+      'path does not match'
+    );
+    assert.equal(
+      ret.method,
+      'GET',
+      'method does not match'
+    );
+    assert.deepEqual(
+      ret.params,
+      { page_size: 300, trash: false, from: defaultDate },
+      'params object does not match'
+    );
+  });
+
+  it('lists User Recordings with pagesize', async function () {
+    const ret = await testAPI.cloudRecording
+      .listUserRecordings({ userId: '12345', pageSize: '39' });
+    assert.deepEqual(
+      ret.params,
+      { page_size: 39, trash: false, from: defaultDate },
+      'params object does not match'
+    );
+  });
+
+
+  it('lists User Recordings with start and end dates', async function () {
+    const ret = await testAPI.cloudRecording.listUserRecordings(
+      {
+        userId: '12345',
+        startDate: '2020/08/20',
+        endDate: '2020-10/5',
+      }
+    );
+    assert.deepEqual(
+      ret.params,
+      {
+        page_size: 300,
+        trash: false,
+        from: '2020-08-20',
+        to: '2020-10-05',
+      },
+      'params object does not match'
+    );
+  });
+
+  it('lists User Recordings with invalid date', async function () {
+    try {
+      await testAPI.cloudRecording.listUserRecordings({ userId: '12345', startDate: 'invalid' });
```

Comment:
If you want to make sure that an error is thrown, you should check that explicitly (this test will pass if no error is thrown, as well)